charts/fybrik/files/taxonomy/data_catalog_request.json

```diff
@@ -0,0 +1,19 @@
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"title": "Data Catalog Request",
+	"type": "object",
+	"properties": {
+	  "datasetid": {
```

Comment:
Is there a naming convention? In go struct it was datasetID

Reply:
changed to assetID. Thanks.